extended-statistics/src/test/java/org/infinispan/stats/logic/OptimisticLockingTxClusterExtendedStatisticLogicTest.java

```diff
@@ -126,7 +124,6 @@ public void testWriteSkewOnOwner() throws Exception {
       doWriteSkewTest(true);
    }
 
-   @Test (groups = "unstable", description = "https://issues.jboss.org/browse/ISPN-3341")
```

Comment:
better, apply this patch to your PR: https://gist.github.com/pruivo/b16f8005135571458b707ad0314be796